src/FilterList.js

```diff
@@ -104,7 +108,9 @@ export let Label = _.flow(
         tree && node && tree.mutate(node.path, { paused: !node.paused })
       }
     >
-      <span {...props} />
+      <span {...props}>
+        {children || (node && _.get([field, 'label'], fields)) || field || ''}
```

Comment:
why do we need to check for `node` before looking up field?

Reply:
Looks like we don't - this was a holdover from when we did `node.field` in the (now dead) FieldLabel component that this line came from.

Just FYI, `node` is actually optional for the Label component if you go by our stories (I think it was in the blueberry story specifically).

Reply:
I just realized that the blueberry story is dead now, and that was the only one that did this - so I guess our stories don't say it's optional after all 🙂